### PR TITLE
Add MongoDB Atlas

### PR DESCRIPTION
**Pull request by myknbani**
They recently added a free tier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -412,6 +412,7 @@ Table of Contents
    * [elephantsql.com](http://www.elephantsql.com/) — PostgreSQL as a service, 20 MB free
    * [graphenedb.com](http://www.graphenedb.com/) — Neo4j as a service, up to 1,000 nodes and 10,000 relations free
    * [mlab.com](https://mlab.com/) — MongoDB as a service, 500 MB free
+   * [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) — free tier gives 512 MB
```
Comment (ripienaar) [inline README.md]: unfortunately I do not see this mentioned on their pricing page clearly, do you have a link that states the terms?

Comment (myknbani) [inline README.md]: It's here:  https://www.mongodb.com/cloud/atlas/pricing

Comment (myknbani) [inline README.md]: This is another:  they did mention that they offer a free tier like mLab

https://www.mongodb.com/cloud/atlas/compare

Comment (ripienaar) [inline README.md]: ah weird, not sure how I missed that - looking again now I think their mobile formating makes it easy to miss, anyway 👍  thanks a lot!

**Pull request merged**